### DISCUSSION TITLE
What does Auto telemetry do?

### DISCUSSION THREAD

**Discussion by AndreKR** [jziolkowski/tdm#252] [Q&A] [answered]
There is a menu item MQTT -> Auto telemetry that can be enabled and disabled. What does it do?

**Comment by barbudor** [comment]
It is issuing a `Status8` command to the devices to retrieve the sensor values. The command is issued at the period specified in MQTT -> Auto-telemetry period menu

However, I can't see anywhere those valyes would be displayed
May be a work in progress

**Reply by jziolkowski** [reply, marked as answer]
This data can be seen in the device's telemetry view. All this function does is trigger sensor data more often than each  device's current setting.
